content/en/APIUseCases/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "API Use Cases"
+linkTitle: "API Use Cases"
+weight: 100
+description: >
+  Includes practical uses for our API.
+---
+
+{{% pageinfo %}}
+The pages in this book describe how you can use our API to manage your Assets
```

Comment:
Since we'll add more guides to this section (not only about assets or pentests), maybe we can make the sentence more generic? Something like:

The pages in this book describe how you can use our API to interact with the Cobalt platform.

The pages in this book describe how you can work with the Cobalt platform using the API.